server/dive_server/views_annotation.py

```diff
@@ -61,7 +77,7 @@ def get_annotations(self, folder):
     )
     def export(self, folder, excludeBelowThreshold: bool, typeFilter: Optional[List[str]]):
```

Comment:
don't know how you feel about maintaining a way to easily download/export a track.json file.  I guess we could always call the `GET dive_annotation/{id}` and direct the response to a JSON file instead.  But it may be nice to have a way to directly download a json for debugging purposes inside a browser (even the swagger tools) instead of relying on command line?  I don't feel too strongly about it if there is a decent reason not to.